packages/gestalt/src/ButtonSocial.css

```diff
@@ -0,0 +1,35 @@
+.social {
+  align-items: center;
+  border-radius: var(--rounding-pill, 999px);
+  cursor: pointer;
+  display: flex;
+  gap: var(--sema-space-200, 8px);
+  height: 48px;
+  padding: var(--Positive-space-300, 12px) var(--Positive-space-1100, 44px)
```

Comment:
this is not a token  var(--Positive-space-300, 12px)

beware Figma specs for tokens are.... not exact ha. 


Make sure VR tokens aren't exposed in classic designs, classic tokens can be exposed in VR cos we map them